<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="WeatherButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Display Temp</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>258</width>
      <height>224</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit2"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit3"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="CityList">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Giza</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cairo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alexandria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hurgada</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="CityList_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Celsius</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fahrenheit</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
